### PR TITLE
Django 1.10 support

### PR DESCRIPTION
**Pull request by gavinwahl**

### CODE REVIEW

File: widgy/models/base.py [hunk 1]
```diff
@@ -839,13 +839,19 @@ def clone(self):
         # See https://code.djangoproject.com/ticket/4027
         new = copy.copy(self)
         unset_pks(new)
+        try:
+            del new._node
+        except AttributeError:
+            pass
         new.save()
+        print((self, self.pk), (new, new.pk))
```
Comment (iainwatts) [inline widgy/models/base.py]: Remove print statement

File: widgy/models/base.py [hunk 2]
```diff
@@ -839,13 +839,19 @@ def clone(self):
         # See https://code.djangoproject.com/ticket/4027
         new = copy.copy(self)
         unset_pks(new)
+        try:
```
Comment (iainwatts) [inline widgy/models/base.py]: Add comment to explain what this does?

**Review by nadzinski — CHANGES_REQUESTED** [review]

**Comment by lockie** [comment]
It would be lovely to have this merged. Anyone?

**Comment by santoshray02** [comment]
I am getting this error

django.db.utils.OperationalError: no such table: django_content_type

**Pull request merged**